tests/test_email_search.py

```diff
@@ -0,0 +1,8 @@
+import unittest
+import os
+import sys
+sys.path.append(os.path.abspath('scripts'))
+
+from outlook_mapi_functions import search_sent_emails
+
+search_sent_emails("welcome")
```

Comment:
These file endings are against pep8. Change accordingly by ensuring that you don't occasionally delete the final newline.